### PR TITLE
Initial Pomodoro Timer

### PR DESCRIPTION
**Pull request by robynsmith**
Hopefully can gather feedback.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+				} else {
+					$(".timer").html("Error: Timer minutes required.");
+				}
+
+				break;
+			case "reset":
+				if (timerIntervalID !== null) {
+					clearInterval(timerIntervalID);
+          timeMin = parseInt($("#min").val());
+          timeSec = 0;
```
Comment (wcarss) [inline main.js:63]: indentation issues all over :P

File: main.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+function test() {
+	console.log("Test");
+}
+
+$(document).ready(function() {
+  timeMin = parseInt($("#min").val());
+  timeSec = 0;
+  cyclesComplete = 0;
+
+	$("button").click(function() {
```
Comment (wcarss) [inline main.js:45]: classes or ids for different button actions and having multiple registered functions would eliminate the need for a switch statement in here

File: main.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		timeMin--;
+	} else {
+		timeSec--;
+	}
+
+	displayString = timeMin + ":" + pad(timeSec, 2);
+
+
+	$(".timer").html(displayString);
+  $("").html(displayString);
```
Comment (wcarss) [inline main.js:26]: what is `$("")`

File: main.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  str = str.toString();
+  return str.length < max ? pad("0" + str, max) : str;
+}
+
+
+function updateTimer() {
+	var displayString = "";
+
+	console.log("Update timer()");
+	if (timeSec === 0) {
```
Comment (wcarss) [inline main.js:15]: could < 0 in case something external modifies it between calls

**Review by wcarss — CHANGES_REQUESTED** [review]
- imo indentation is easy to fix and important even in your own code
- global vars ew, but, understandable and probablyt he simplest option for this use case
- cool story bro

**Comment by wcarss** [comment]
booo closed without merge bad maintenance

**Comment by robynsmith** [comment]
@wcarss The lulz.

Just clearing out my "Pull Requests" page of things I'm not actively working on.

**Pull request closed without merging**